csharp/Directory.Build.props

```diff
@@ -29,7 +29,7 @@
     <Product>Apache Arrow ADBC library</Product>
     <Copyright>Copyright 2022-2024 The Apache Software Foundation</Copyright>
     <Company>The Apache Software Foundation</Company>
-    <Version>1.0.0</Version>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
if we also add a VersionSuffix of SNAPSHOT, would that make builds from the main branch produce versions of 1.0.0-SNAPSHOT when the version suffix is not overridden? If so, this seems like a good change to make at the same time. It would be consistent with a bunch of the other builds.

Reply:
Change made, I've added the default version suffix of "SNAPSHOT" to the Props file.
This means that the default package version is _0.13.0-SNAPSHOT_.